app/models/user.rb

```diff
@@ -15,8 +15,11 @@ class User < ActiveRecord::Base
   has_many :grade_entry_students
   has_many :groupings, through: :memberships
   has_many :notes, as: :noteable, dependent: :destroy
-  has_many :accepted_memberships, class_name: 'Membership', conditions: {membership_status: [StudentMembership::STATUSES[:accepted], StudentMembership::STATUSES[:inviter]]}
-
+  has_many :accepted_memberships,
+           -> { where membership_status: 
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.<br>Trailing whitespace detected.